mail_sendgrid/__init__.py

```diff
@@ -0,0 +1,14 @@
+# -*- encoding: utf-8 -*-
```

Comment:
Please use current template and put correctly `coding`: https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L1-L3